.gitignore

```diff
@@ -17,4 +17,5 @@ package-lock.json
 **/.history
 
 # Exception for extension recommendations
-!**/.vscode/extensions.json
\ No newline at end of file
+!**/.vscode/extensions.json
+/.vs
```

Comment:
Delete it to revert .gitignore to its state in repo. You shouldn't change this file

Reply:
I reverted it to its original state. Sorry. That was my VS19 messing